README.md

```diff
@@ -43,6 +59,14 @@ Putting it all together
 curl -LSs https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh -O install.sh
 INSTALL_PREFIX=/tmp/t1 LUNARVIM_CONFIG_DIR=/tmp/t2 LUNARVIM_RUNTIME_DIR=/tmp/t3 bash ./install.sh
 ```
+for Windows use
+``` powershell
+$LVBRANCH=rolling; (Invoke-WebRequest "https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.ps1" -usebasicparsing).Content | powershell
```

Comment:
Use `$LV_BRANCH` variable in README like in `install.ps1`.
```suggestion
$LV_BRANCH=rolling; (Invoke-WebRequest "https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.ps1" -usebasicparsing).Content | powershell
```

Reply:
This actually needs to be updated in several locations, but I figured for cleanliness I'd only add it to one.

Also, there seem to be two copies of the Windows installation script in the README for both standard and custom installation. Is there any reason for that?

Reply:
> This actually needs to be updated in several locations, but I figured for cleanliness I'd only add it to one.
I'll check it

>Also, there seem to be two copies of the Windows installation script in the README for both standard and custom installation. Is there any reason for that?

No valid reason. This is still an open conflict. I will address that.